README.md

```diff
@@ -112,3 +112,14 @@ Run the following commands:
 docker build --target test --tag todo-app:test .
 docker run todo-app:test
 ```
+
+## Deployement process
+
+run
+
+```
+docker build --target <build_phase> --tag aqsasaied/todo-app:prod .
```

Comment:
(not important) It seems a little strange that you've hardcoded the `prod` tag here but templated out the `<build_phase>`